com.pinegrow.Pinegrow.yaml

```diff
@@ -0,0 +1,45 @@
+app-id: com.pinegrow.Pinegrow
+runtime: org.freedesktop.Sdk
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: pinegrow
+separate-locales: false
+finish-args:
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+modules:
+
+  - name: pinegrow
+    buildsystem: simple
+    build-commands:
+      - install -D apply_extra /app/bin/apply_extra
+      - install -D pinegrow /app/bin/pinegrow
+      - install -Dm644 -t /app/share/metainfo com.pinegrow.Pinegrow.appdata.xml
+      - install -Dm644 -t /app/share/applications com.pinegrow.Pinegrow.desktop
+      - install -Dm644 -t /app/share/icons/hicolor/scalable/apps com.pinegrow.Pinegrow.svg
+      - install -d "${FLATPAK_DEST}/extensions"
+      - install -d "${FLATPAK_DEST}/packages"
+    sources:
+      - type: extra-data
+        url: https://github.com/Pinegrow/PinegrowReleases/releases/download/pg7.4/PinegrowLinux64.7.4.zip
```

Comment:
is extra-data necessary? There is nothing that same to indicate it's not redistributable.

also you still build an aarch64 flatpak, which I'll assume will not work since this is a binary release. Or it's x86_64 that will not (unlikely)

Reply:
still the first question is left unanswered.

Reply:
> is extra-data necessary? There is nothing that same to indicate it's not redistributable.
> still the first question is left unanswered.

Not sure what that would imply or if i got your question. How would i make sure of that ?

Reply:
extra_data is meant to allow packaging software that can't be redistributed, making the download part of the user installation process, at a price of less reliability (ie if it doesn't exist anymore or change it won't work).

In general if you can redistribute the binaries, don't use extra-data.

Reply:
> extra_data is meant to allow packaging software that can't be redistributed, making the download part of the user installation process, at a price of less reliability (ie if it doesn't exist anymore or change it won't work).
> 
> In general if you can redistribute the binaries, don't use extra-data.

Ok so I should just change the type to archive, right ? Anything else would need to change ?

Reply:
> > extra_data is meant to allow packaging software that can't be redistributed, making the download part of the user installation process, at a price of less reliability (ie if it doesn't exist anymore or change it won't work).
> > In general if you can redistribute the binaries, don't use extra-data.

Quote from my email exchange with Pinegrow "I've read about it a while
ago and thought that it would be a very useful way to distribute
Pinegrow on Linux. The best point is that the app files are always
downloaded from the official sources."

I would tend to understand that as them preferring it to be downloaded from their servers, what do you think ?